### PR TITLE
Make it possible to automatically deploy the newly built Kafka Connec…

### PR DESCRIPTION
**Pull request by scholzj**
…t images #45

### CODE REVIEW

File: kafka-connect/s2i/resources/openshift-template.yaml [hunk 1]
```diff
@@ -36,6 +61,9 @@ objects:
     labels:
       build: ${BUILD_NAME}
     name: ${TARGET_IMAGE_NAME}
+  spec:
+    lookupPolicy:
+      local: true
```
Comment (tombentley) [inline kafka-connect/s2i/resources/openshift-template.yaml]: What does this do?

File: kafka-connect/s2i/resources/openshift-template.yaml [hunk 2]
```diff
@@ -3,6 +3,31 @@ kind: Template
...
+  displayName: Number of Kafka connect instances
+  name: INSTANCES
+  required: true
+  value: "3"
+- description: A list of host:port pairs to use for establishing the initial connection to the Kafka cluster.
+  displayName: Kafka bootstrap servers
+  name: KAFKA_CONNECT_BOOTSTRAP_SERVERS
+  required: true
+  value: kafka:9092
+- description: A unique string that identifies the Connect cluster group this worker belongs to. Note this must not conflict with any consumer group IDs.
```
Comment (tombentley) [inline kafka-connect/s2i/resources/openshift-template.yaml]: long line

Comment (scholzj) [reply kafka-connect/s2i/resources/openshift-template.yaml]: Fixed. At least assuming you meant it as "too long in the YAML file".

File: kafka-connect/s2i/resources/openshift-template.yaml (hunk 1 above)
Comment (scholzj) [reply kafka-connect/s2i/resources/openshift-template.yaml]: This should allow you to use the image from Kubernetes resources (such as Deployment or StateFulSet) without specifying the registry URL.

**Comment by tombentley** [comment]
lgtm.

**Pull request merged**